### DISCUSSION TITLE
Sensor

### DISCUSSION THREAD

**Discussion by im-trisha** [halildurmus/win32#493] [Q&A] [answered]
I really hate to ask dumb questions... But I really can't find it
I was trying a simple snippet from msdn
```
var hr = CoCreateInstance(CLSID_SensorManager, 
                        nullptr, CLSCTX_, INPROC_SERVER,
                        IID_PPV_ARGS(&pSensorManager));

if(hr == HRESULT_FROM_WIN32(ERROR_ACCESS_DISABLED_BY_POLICY)){}
```
The CLSID_SensorManager GUID doesn't seem to be there at all.

Does this library supports windows sensors?

PS: A bit "Off topic" but i don't seem to find a way to run a .exe as a service (No window)

**Comment by halildurmus** [comment, marked as answer]
> Does this library supports windows sensors?

Sensor API is not supported yet. I've filed #494 for this.

> PS: A bit "Off topic" but i don't seem to find a way to run a .exe as a service (No window)

There are programs that let you run any executable (not just Dart .exe) as a Windows service: [winsw](https://github.com/winsw/winsw), [NSSM](http://nssm.cc)

I tried `winsw` and it works.

**Reply by im-trisha** [reply]
Thanks!
